MdeModulePkg/Library/UefiBootManagerLib/BmConsole.c

```diff
@@ -432,6 +432,9 @@ EfiBootManagerUpdateConsoleVariable (
   EFI_STATUS                Status = EFI_SUCCESS;    // MU_CHANGE
   EFI_DEVICE_PATH_PROTOCOL  *VarConsole;
   EFI_DEVICE_PATH_PROTOCOL  *NewDevicePath;
+  // MU_CHANGE [BEGIN] - Explicitly initialize pointers to NULL

```

Comment:
Consider using the other comment tag for this?  `MU_CHANGE [BEGIN] - Check new device path before save`

Reply:
I agree that makes more sense because this is a new pointer.

Reply:
Done in PR commit https://github.com/microsoft/mu_basecore/pull/445/commits/ef2591349b8ca8cee542cea979133a9b88226bcf